### PR TITLE
Upgrade to Angular 8 #LMR-1770

### PR DESCRIPTION
**Pull request by livthomas**

### CODE REVIEW

File: src/app/core/store/organizations/payment/payments.effects.ts [hunk 1]
```diff
@@ -131,9 +129,7 @@ export class PaymentsEffects {
   public createPayment$: Observable<Action> = this.actions$.pipe(
     ofType<PaymentsAction.CreatePayment>(PaymentsActionType.CREATE_PAYMENT),
     mergeMap(action => {
-      const returnUrl =
-        (action.payload.returnUrl && action.payload.returnUrl) ||
-        (this.location as BrowserPlatformLocation).location.href;
+      const returnUrl = (action.payload.returnUrl && action.payload.returnUrl) || window.location.href;
```
Comment (livthomas) [inline src/app/core/store/organizations/payment/payments.effects.ts]: @marvec Is this what you have tried to accomplish here?

Comment (marvec) [reply src/app/core/store/organizations/payment/payments.effects.ts]: Smeg knows ;-) Looks so!

Comment (kubedo8) [reply src/app/core/store/organizations/payment/payments.effects.ts]: There is redundant `action.payload.returnUrl`

Comment (marvec) [reply src/app/core/store/organizations/payment/payments.effects.ts]: I see, the first should have been `action.payload` to see if it was there.

Comment (livthomas) [reply src/app/core/store/organizations/payment/payments.effects.ts]: Oh, I see. But it's always non-null so there is no need to check it. TypeScript would complain about it if you created a new instance of `CreatePayment` class without a payload. We rely on it in all actions.

**Pull request merged**